### PR TITLE
Module2 purchase in online store

### PR DESCRIPTION
**Pull request by Viktor24500**
Task
Make a purchase in the online store.

Criteria
--- Get an array of some products
--- Select from the array several products for purchase (up to 10 count) and add to the shopping cart.
--- The shopping cart stores items until an order has been placed.
--- Place an order for goods
--- Upon registration, it is necessary to notify the buyer that an order with a specific number has been generated

Notes
--- App architecture should be flexible & and the similar with this
--- App must have 1 entrypoint for all classes (Services and Repositories)

### CODE REVIEW

File: Module2_HW2/Module2_HW2/Store.cs [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+        }
+
+        public void ReviewShopingCart()
+        {
+            _shoppingCart.DisplayShopingCart();
+        }
+
+        private void AddDataToStorage()
+        {
+            Product product1 = new Product("keyboard", 10);
```
Comment (YevhenBuriak) [inline Module2_HW2/Module2_HW2/Store.cs:101]: Why initialization with data is a part of that class? What happens if I want to change the initialization list (add other products)?

File: Module2_HW2/Module2_HW2/Store.cs [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+            AddToStorage(product3);
+            AddToStorage(product4);
+            AddToStorage(product5);
+            AddToStorage(product6);
+            AddToStorage(product7);
+        }
+
+        private class ShopingCart
+        {
+            private Store _store;
```
Comment (YevhenBuriak) [inline Module2_HW2/Module2_HW2/Store.cs:120]: Do not see where this field is initialized. How is it used?

**Review by YevhenBuriak — APPROVED** [review]
The logic for storing products could be moved to a separate repository. Did not get how you use _store field in the Cart service do not see what actually inits it.

File: Module2_HW2/Module2_HW2/Store.cs (hunk 2 above)
Comment (Viktor24500) [reply Module2_HW2/Module2_HW2/Store.cs:120]: I use _store for calling methods from public class Store

**Pull request merged**